crashtracker/src/crash_info/metadata.rs

```diff
@@ -3,7 +3,7 @@
 use ddcommon::tag::Tag;
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, Default, Clone, PartialEq, Serialize, Deserialize)]
 pub struct CrashtrackerMetadata {
```

Comment:
I do not think we would want this to have empty defaults. Any reason to allow this ?

Reply:
Why not if there are already defaults outside of Serde and they work 🤔

Reply:
What I meant is that you would want the user to fill some of these fields (like family). Feel free to ignore I'm not sure how you would make that field mandatory.

Reply:
That's the main thing. I think a configuration object struct should accept basically anything, and it should be up to the implementation using it to validate if the fields have values that are valid for whatever class using the object. In that sense, I think having a default value for every field makes sense, and simplifies usage by language-specific bindings.

Reply:
I disagree, in this case the default value is not valid and we should make it invalid by construction rather than validation.
For instance `family` really should be an enum, although right now it's more practical for it to be a String.
People should think about what they want in their metadata and deriving Default automatically goes against that

Reply:
I was replying basically for all 3 objects, but for `CrashtrackerMetadata` I tend to agree that most fields are mandatory, I added `Default` basically because I was doing it also for the other ones. I'll change that one so that only tags are optional.